src/room/track/RemoteTrackPublication.ts

```diff
@@ -67,6 +77,11 @@ export default class RemoteTrackPublication extends TrackPublication {
     return super.isSubscribed;
   }
 
+  // returns client's desire to subscribe to a track, also true if autoSubscribe is enabled
+  get isDesired(): boolean {
+    return !!this.subscribed;
```

Comment:
is this equivalent to 
```suggestion
    return this.subscribed !== false;
```

`!!` is always harder for me to understand.. 😅